tidb-cloud/naming-conventions-for-data-import.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Naming Conventions for Data Import
+summary: Learn about the naming conventions for CSV, Parquet, Aurora Snapshot, and SQL files during data import.
+---
+
+# Naming Conventions for Data Import
+
+The Data Import feature in TiDB Cloud supports the following file formats: CSV, Parquet, Aurora Snapshot, and SQL. To make sure that data can be imported successfully, you need to prepare the following:
+
+- Prepare the SQL file for creating the target database (optional) and the SQL file for creating the target table that conforms to the naming conventions. If the SQL file for creating the target table is not provided, you need to create the corresponding table manually in the target database in advance.
+- Prepare a data file that conforms to the naming conventions for importing data. If the data file name cannot meet the requirements, it is recommended to use the file pattern to perform the import task. Otherwise, the import task can not scan the data files you want to import.
+
+## Naming conventions for schema files
+
+This section describes the naming conventions for database and table schema files. The naming conventions for schema files are the same for all the following types of source files: CSV, Parquet, Aurora Snapshot, and SQL.
+
+The naming conventions for schema files are as follows:
+
+- Database schema file(optional): `${db_name}-schema-create.sql`
+- Table schema file: `${db_name}.${table_name}-schema.sql`
+
+For example:
+
+- mytestdb-schema-create.sql
+- mytestdb.testtable-schema.sql
+
```

Comment:
1. In the examples below, the db name is `import_db` and the table name is `test_table`.  Do we need to unify the name here also ?
2. Do we need to also give an example of the file content? Like this: 

Database schema file name: `import_db-schema-create.sql`
File content example: 
```sql
CREATE DATABASE import_db;
```

Table schema file name: `import_db.test_table-schema.sql`
File content example: 
```sql
CREATE TABLE test_table (
    id INTEGER PRIMARY KEY,
    val VARCHAR(255)
);
```

Reply:
Good suggestion. Updated.